spec/controllers/families_controller_spec.rb

```diff
@@ -0,0 +1,28 @@
+require "rails_helper"
+
+RSpec.describe FamiliesController do
+  describe "POST /families" do
+    context "when invalid params" do
+      it "doesn't create a new family" do
+        expect do
+          post :create, params: { family: { first_name: "John" } }
+        end.not_to change(Family, :count)
+      end
+
+      it "flashes an alert message and render the new family template again" do
+        post :create, params: { family: { first_name: "John" } }
+
+        expect(flash[:alert])
+          .to match(
+            I18n.t("flash.actions.create.alert", resource_name: "Family")
+          )
```

Comment:
You will probably need a translation for your resource_name (I figure I will need to do the same lol)